slick/codegen/src/main/scala/com.workday.warp/TableCodeGenerator.scala

```diff
@@ -170,14 +170,16 @@ implicit object ${wrapperName}TypeClassObject extends $parentTypeClassName[$wrap
             indent("object RowTypeClasses {\n" + tables.map(generateTypeClass).mkString("\n")) +
             "\n}" +
             "\n\n" +
-            tables.map(_.code.mkString("\n")).mkString("\n\n")
+            tables.map(_.code.mkString("\n")).mkString("\n\n") +
+            "\n\n"
         }
 
         // Wraps the code with a trait "Tables" and the correct package.
         override def packageCode(profile: String, pkg: String, container: String, parentType: Option[String]): String = {
           s"""
 package ${pkg}
 // !!! AUTO-GENERATED Slick data model, do not modify.
+// scalastyle:off
```

Comment:
unfortunately, the new scalastyle can only be configured via gradle project `sourceSets` and no place to set actual directories like the previous scalastyle: https://github.com/alisiikh/gradle-scalastyle-plugin/blob/master/src/main/groovy/com/github/alisiikh/scalastyle/ScalastyleExtension.groovy#L88

Alternatively, we can declare our generated code into a separate source set and compile that as a dependency in our main/test source sets. I'm up for either option